.github/workflows/build.yaml

```
@@ -44,23 +56,25 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-nextjs-${{ hashFiles('**/pnpm-lock.yaml') }}-
 
+      - name: Generate Build Markdown
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2037:error:1:1: To assign the output of a command, use var=$(cmd) 

(shellcheck)